spring-boot/spring-boot-elasticsearch/README.md

```diff
@@ -0,0 +1,5 @@
+# Related Blog Posts
```

Comment:
It would be great if the README contained a quick info on how to start the application (i.e. start the elasticsearch docker image, then start the Spring Boot app, then open this URL in your browser, then type one of these search queries into the field)